rerun_py/rerun_sdk/rerun/_log.py

```diff
@@ -43,6 +43,9 @@ def component_name(self) -> str:
     def as_arrow_array(self) -> pa.Array:
         return self.data
 
+    def component_descriptor(self) -> ComponentDescriptor:
+        return ComponentDescriptor(self.component_name())
```

Comment:
Not yet -- because other SDKs don't and therefore roundtrips would fail.

But yes, for milestone 2 this will be the case, so that our backwards compat plan can work.